packages/turf-line-chunk/index.js

```diff
@@ -32,30 +31,21 @@ module.exports = function (featureIn, segmentLength, units, reverse) {
     var outFeatures = [];
     var debug = arguments['4']; // Hidden @param {boolean} Enable debug mode
 
-    // Handles FeatureCollection
-    featureEach(featureIn, function (multiFeature) {
-
-        // Handles MultiLineString
-        if (multiFeature.geometry.type === 'MultiLineString') {
-            multiFeature = flatten(multiFeature);
+    // Flatten each feature to simple LineString
+    flattenEach(featureIn, function (feature) {
+        if (reverse) {
+            feature.geometry.coordinates = feature.geometry.coordinates.reverse();
         }
-
-        // All features are simple LineString
-        featureEach(multiFeature, function (feature) {
-            if (reverse) {
-                feature.geometry.coordinates = feature.geometry.coordinates.reverse();
+        var lineSegments = sliceLineSegments(feature, segmentLength, units);
+        lineSegments.forEach(function (segment, index) {
+            if (debug === true) {
```

Comment:
Didn't notice `(debug===true)` was still here, this section can be removed since it can easily be handled in the `test.js`.